src/Pages/Repositories/ContentListTable/ContentListTable.tsx

```diff
@@ -227,6 +227,11 @@ const ContentListTable = () => {
     sortString,
   );
 
+  const triggerIntrospectionAndSnapshot = (repoUuid) => {
+    introspectRepoForUuid(repoUuid).then(clearCheckedRepositories);
```

Comment:
this won't run serially, they will be running parallel. Is that intended?

Reply:
fixed this and the above :) let me know if you see anything else